File: specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/examples-java/DeleteStudent.java
import com.azure.core.util.Context;

/** Samples for Students Delete. */
public final class Main {
    /*
     * x-ms-original-file: specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/examples/DeleteStudent.json
     */
    /**
     * Sample code: DeleteLab.
     *
     * @param manager Entry point to EducationManager.
     */
    public static void deleteLab(com.azure.resourcemanager.education.EducationManager manager) {
        manager
            .students()
            .deleteWithResponse(
                "{billingAccountName}", "{billingProfileName}", "{invoiceSectionName}", "{studentAlias}", Context.NONE);
    }
}

File: specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/examples-js/DeleteStudent.js
const { EducationManagementClient } = require("@azure/arm-education");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Delete the specified student based on the student alias.
 *
 * @summary Delete the specified student based on the student alias.
 * x-ms-original-file: specification/education/resource-manager/Microsoft.Education/preview/2021-12-01-preview/examples/DeleteStudent.json
 */
async function deleteLab() {
  const billingAccountName = "{billingAccountName}";
  const billingProfileName = "{billingProfileName}";
  const invoiceSectionName = "{invoiceSectionName}";
  const studentAlias = "{studentAlias}";
  const credential = new DefaultAzureCredential();
  const client = new EducationManagementClient(credential);
  const result = await client.students.delete(
    billingAccountName,
    billingProfileName,
    invoiceSectionName,
    studentAlias
  );
  console.log(result);
}
